examples/interaction-shortcut/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/shomali11/slacker/v2"
+	"github.com/slack-go/slack"
+)
+
+// Implements a basic interactive command with modal view.
+
+func main() {
+	bot := slacker.NewClient(
+		os.Getenv("SLACK_BOT_TOKEN"),
+		os.Getenv("SLACK_APP_TOKEN"),
+		slacker.WithDebug(false),
+	)
+
+	bot.AddInteraction(&slacker.InteractionDefinition{
+		InteractionID: "mood-survey-message-shortcut-callback-id",
+		Handler:       moodShortcutHandler,
+		Type:          slack.InteractionTypeMessageAction,
+	})
+
+	bot.AddInteraction(&slacker.InteractionDefinition{
+		InteractionID: "mood-survey-global-shortcut-callback-id",
+		Handler:       moodShortcutHandler,
+		Type:          slack.InteractionTypeShortcut,
+	})
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	err := bot.Listen(ctx)
```

Comment:
Does this file do anything? How is the interaction getting initiated without a command?

Reply:
It's a shortcut at the message level or global:
<img width="1726" alt="image" src="https://github.com/shomali11/slacker/assets/60900917/841b238a-3f69-4c19-8453-e1a3caf2b5b4">
<img width="1726" alt="image" src="https://github.com/shomali11/slacker/assets/60900917/52613a13-722f-4fe7-a2e8-f875800aec48">